source-gallery/src/main/java/com/google/android/apps/muzei/gallery/GallerySettingsActivity.java

```diff
@@ -356,34 +354,8 @@ public boolean onCreateOptionsMenu(final Menu menu) {
     @Override
     public boolean onPrepareOptionsMenu(final Menu menu) {
         super.onPrepareOptionsMenu(menu);
-        // Make sure the 'Import photos' MenuItem is set up properly based on the number of
```

Comment:
It feels like this comment should still be here, rather than down in the other method.

Reply:
OK

Reply:
Done